### ISSUE TITLE
How to check hash key of the app when published on playstore?

### ISSUE THREAD

**Issue by DevHitesh29** [jaumard/sms_autofill#161]
I checked the hash key on debug and release mode. The haskey is different for each mode and might differ in the app bundle. So how can I get the haskey of the build which is published on the play store so that autofill functionality can be resumed by changing the key on the SMS template?

Thank you in advance

**Comment by kashiflab** [comment]
@DevHitesh29 did you find how to get the release mode hash key?

**Comment by danishkhawaja98** [comment]
Hi @DevHitesh29 
Is the app signature/hash being same for android **_release APK_** and **_app  bundle_**?

**Comment by Hiteshc29** [comment]
> @DevHitesh29 did you find how to get the release mode hash key?

Yes implemented a workaround to get the hash key on release mode by printing it on the snack bar and removing the code later in the final release.

**Comment by Hiteshc29** [comment]
> Hi @DevHitesh29
> Is the app signature/hash being same for android **_release APK_** and **_app bundle_**?

Yes they are same

**Comment by danishkhawaja98** [comment]
> > Hi @DevHitesh29
> > Is the app signature/hash being same for android **_release APK_** and **_app bundle_**?
> 
> Yes they are same

According to this [article](https://prishanmaduka.medium.com/generate-hash-keys-properly-for-android-sms-retriever-api-c26b9be42ddc), the app signature for release build and signed APK are different.
Can you look into this and explain me that what I am doing wrong.
